crates/pop-cli/src/commands/new/mod.rs

```diff
@@ -62,6 +64,80 @@ impl Display for Command {
 	}
 }
 
+/// Guide the user to select what type of project to create
+pub async fn guide_user_to_select_command() -> AnyhowResult<Command> {
+	Cli.intro("🚀 Welcome to Pop CLI!")?;
+	
+	let mut prompt = cliclack::select("What would you like to create?".to_string());
+	
+	// Add available options based on features
+	#[cfg(feature = "parachain")]
+	{
+		prompt = prompt.item(
+			"parachain", 
+			"🌐 Parachain", 
+			"Build your own blockchain with custom logic, token economics, and governance"
+		);
+		prompt = prompt.item(
+			"pallet", 
+			"🧩 Pallet", 
+			"Create reusable blockchain modules to add features like tokens, voting, or custom logic"
```

Comment:
Same as above, I wouldn't be that specific with the examples. Shorter:
`Create reusable and customizable chain modules`